src/R/Components/Test/Configuration/ConfigurationSettingsCollectionTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using FluentAssertions;
+using Microsoft.R.Components.Application.Configuration;
+using Microsoft.UnitTests.Core.XUnit;
+
+namespace Microsoft.R.Components.Test.Configuration {
+    [ExcludeFromCodeCoverage]
+    [Category.Configuration]
+    public class ConfigurationSettingsCollectionTest {
+        [Test]
+        public void Test01() {
+            string file = Path.GetTempFileName();
```

Comment:
Let's use `TestFilesFixture` and create file in a test folder - in this case we will be able to verify that file in case of test failing